### ISSUE TITLE
Enable JupyterLab source maps when developing federated extensions

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/jupyterlab#9238]
When debugging a third-party *federated* extension, extension authors can already enable source maps for their extension by running:

```
jupyter labextension build --development True .
```

However it is often useful to also be able to step into JupyterLab core extensions. At the moment extension authors are presented with the minified assets from lab:

![image](https://user-images.githubusercontent.com/591645/97464951-d14d3a00-1941-11eb-987c-fa52e1d4a9e8.png)

It would be nice to have it enabled when building the federated extension in dev mode (with `jupyter labextension build --development True`). Alternatively this could be documented in the cookiecutter or the JupyterLab developer documentation.

**Comment by jtpio** [comment]
For now it's possible to generate the source maps by running:

```
jupyter lab build --minimize=False
```

once before starting JupyterLab.

**Comment by jasongrout** [comment]
> Alternatively this could be documented in the cookiecutter or the JupyterLab developer documentation.

I think the easiest thing for 3.0 would be to document your `jupyter lab build --minimize=False` tip above.

**Comment by jtpio** [comment]
Added in https://github.com/jupyterlab/extension-cookiecutter-ts/pull/110, since for now this is where the `jupyter labextension build --development True .` command is.

**Comment by jtpio** [comment]
We can consider adding it to the JupyterLab documentation later if needed.

**Comment by jasongrout** [comment]
Thanks!
